### ISSUE TITLE
NPM Modules under organizations are being bundled into modules.js

### ISSUE THREAD

**Issue by abhiaiyer91** [meteor/meteor#7640]
@benjamn At workpop we were using version 1.3.2.4 and we ran `meteor build` we examined the tarball and found our npm modules under the workpop organization where we expect:

`programs/server/npm/node_modules`

but having upgraded to Meteor 1.4, these modules are now found in `packages/modules.js`

This causes an interesting bug:

At workpop we have a node_module called us-city-reverse-geocode. This module is currently reading a CSV and the build tool is confusing the path. When logging the path the dirname 

`/node_modules/@workpop/us-city-reverse-geocode/src`
which is incorrect. it is taking the root of my filesystem not of my project directory.

Whats going on here!? The code below is found in one of those organization modules.
```
var filePath = path.join(__dirname, './cityzip.csv');
fs.createReadStream(filePath).pipe(parser);
```

**Comment by benjamn** [comment]
Meteor builds a bundle of server node_modules (`bundle/programs/server/packages/modules.js`) just like it does for client modules, as a fallback in case it's not able to use native Node to run server code in `bundle/programs/server/npm/node_modules`.

If all goes well, that bundle of server modules will never be used, but it turns out there's a bug in 1.4 that prevents @-scoped modules like `@workpop/us-city-reverse-geocode` from being copied into `programs/server/npm/node_modules`, so your app is falling back to the server bundle, which has certain limitations compared to Node, e.g. `__dirname` will be relative to the app (rather than the file system root) and the bundle only contains imported files, so `cities.csv` probably won't be available at runtime.

Given your use of @-scoped packages and general desire to use a stable version of Meteor, I would recommend waiting to upgrade to 1.4.1, which [fixes this problem](https://github.com/meteor/meteor/pull/7609). Running `meteor --release 1.4.1-rc.0 deploy ...` might work in the meantime.

**Comment by benjamn** [comment]
This should be fixed if you `meteor update` to 1.4.1. Feel free to reopen if that's not the case!
